index.html

```diff
@@ -517,6 +521,10 @@ <h1>Terminology</h1>
 	<!--  USE CASES -->
 	<section id="sec-use-cases" class="informative">
 		<h1>Use Cases</h1>
+		<p>
```

Comment:
To make a section non-normative, just add `class="informative"` for the section tag.
ReSpec automatically generate `<em>This section is non-normative.</em>` paragraph.
(see 'Section' section in [ReSpec Editor's Guide](https://github.com/w3c/respec/wiki/ReSpec-Editor's-Guide#sections))

Reply:
@k-toumura thanks for the tip. I will merge this PR and update directly in the master branch.